addon/mixins/file-cache-bypass.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+    fileManager: Ember.inject.service(),
+
+    ajaxOptions() {
+        // When reloading file models after a file action, bypass the cache
+        let options = this._super(...arguments);
+        if (this.get('fileManager.isReloading')) {
+            let prefix = options.url.indexOf('?') === -1 ? '?' : '&';
+            let nonce = Date.now();
+            options.url += `${prefix}cachebypass=${nonce}`;
```

Comment:
A docstring here might be helpful for posterity.

As clarified by Abram: this is not a documented behavior triggered by specific API code. Rather, adding *anything* unique to the URL will force varnish to bypass the cache.